### PR TITLE
Allow polls to have anonymous votes

### PR DESCRIPTION
**Pull request by jszanto**
This is required in order to migrate polls from the previous GEWIS website.

### CODE REVIEW

File: module/Frontpage/src/Frontpage/Model/PollOption.php [hunk 1]
```diff
@@ -116,13 +123,21 @@ public function setEnglishText($englishText)
...
+
+    /**
      * Get the number of votes for this poll option.
      *
      * @return integer
      */
     public function getVotesCount()
     {
-        return is_null($this->votes) ? null : $this->votes->count();
+        return $this->anonymousVotes + (is_null($this->votes) ? null : $this->votes->count());
```
Comment (stefanmajoor) [inline module/Frontpage/src/Frontpage/Model/PollOption.php]: null should be 0

Comment (jszanto) [inline module/Frontpage/src/Frontpage/Model/PollOption.php]: Fixed.

**Comment by kokx** [comment]
Isn't this solution way too hacky? Shouldn't we have separate `PollVote` instances for every vote?

**Comment by jszanto** [comment]
@kokx `PollVote` instances must be bound to a user id to prevent duplicate voting. However there is no way we can do this when this data is missing.

**Comment by kokx** [comment]
Well, in that case we don't really have any other option indeed. Thanks for the explanation!

**Pull request merged**